### PR TITLE
[SRM-512] SDPAP-6730 Remove deprecated "drupal_get_path()"

### PR DESCRIPTION
**Pull request by mayurngondhkar**
**Jira:**
https://digital-vic.atlassian.net/browse/SRM-512
https://digital-vic.atlassian.net/browse/SDPAP-6730

**Changes**

1. Remove deprecated `drupal_get_path()` function

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
-        "dpc-sdp/tide_core": "^3.0.0",
-        "dpc-sdp/tide_landing_page": "^3.0.0"
+        "dpc-sdp/tide_core": "dev-feature/SRM-512_update_drupal_to_9.4",
+        "dpc-sdp/tide_landing_page": "dev-feature/SRM-512_update_drupal_to_9.4"
```
Comment (MdNadimHossain) [inline composer.json]: revert it before merging

File: dev-tools.sh [hunk 2]
```diff
@@ -33,6 +33,6 @@
 # stable version.
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
-# export GH_COMMIT=COMMIT_SHA
+export GH_COMMIT=9dbb1773923a6b5ac95583a21cf1b293a593679a
```
Comment (MdNadimHossain) [inline dev-tools.sh]: revert it before merging

**Review by MdNadimHossain — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (mayurngondhkar) [reply composer.json]: Changes reverted

File: dev-tools.sh (hunk 2 above)
Comment (mayurngondhkar) [reply dev-tools.sh]: Changes reverted

**Pull request merged**